src/core/platforms/android/androidplatformutilities.cpp

```diff
@@ -470,7 +470,7 @@ QJniObject AndroidPlatformUtilities::getNativeExtras() const
 
 ResourceSource *AndroidPlatformUtilities::processCameraActivity( const QString &prefix, const QString &filePath, const QString &suffix, bool isVideo )
 {
-  if ( !checkCameraPermissions() )
+  if ( !triggerCameraPermissions() )
```

Comment:
Does this "trigger a permission", "trigger a permission request" or "request a permission"? (proposal: `requestCameraPermission`)

Reply:
@m-kuhn , well, it checks and if needed requests. 

The reason why I went through this renaming is that I wanted to get to a function name that is distinct from  what Qt >= 6.5 adopted.

Starting with Qt 6.5, QCoreApplication has a checkPermission( type ) and a requestPermission( type, function ), which we parallel in the new functions added to PlatformUtilities (i.e. check{Camera,Microphone}Permission, request{Camera,Microphone}Permission).

To avoid having a confusing checkCameraPermissions() that checks and requests vs. checkCameraPermission that only checks, I renamed the former.